jetstream.md

```diff
@@ -254,6 +254,28 @@ setInterval(() => {
 Note the above example is contrived, as the pull interval is fixed based on some
 interval.
 
+#### JetStream Queue Consumers
+
+Queue Consumers allow scaling the processing of messages stored in a stream. To
+create a Queue Consumer, you have to set its `deliver_group` property to the
+name of the queue group (or use the `ConsumerOptsBuilder#queue()`. Then reuse
```

Comment:
I think you have a missing `)`, to close the "( or use the..".

Reply:
Thanks Ivan - my eyes are not focusing right.